static/js/settings.js

```diff
@@ -0,0 +1,43 @@
+//DOM
+const save_settings = document.getElementById('save_settings')
+const fire = document.getElementById('fire')
+const activate_modals = document.getElementsByClassName('activate_modal')
+
+//GLOBAL
+state = 0
+
+//MODALS - Event Listener 
+for (const button of activate_modals) {
+    button.addEventListener('click', function(e) {
+        const type = e.target.id
```

Comment:
This is a little confusing to me. You are attaching a click handler to an element and then you are using the id to find elements with the same class name and setting them as visible. Does this mean you have a dom structure like:


```
<button id="user-modal">Click Me</button>
<div class="user-modal" style={{ visibility: 'none' }}>When clicked this is visible</div>
```

This might just be because I am use to UI frameworks and this is very barebone Javascript...